packages/stores/types.ts

```diff
@@ -308,16 +309,17 @@ type TUiStore = {
     setShouldShowCooldownModal: (value: boolean) => void;
 };
 
-type TPortfolioStore = {
-    active_positions: ProposalOpenContract[];
-    error: TCommonStoreError;
-    getPositionById: (id: number) => ProposalOpenContract;
+export type TPortfolioStore = {
```

Comment:
@farzin-deriv  Could you please suggest where it's better to put `TPortfolioStore` so that I can use it in open-positions file in `reports` package 🙂 ? will it be ok if I put it in `shared`? But then I will need to import it from `shared` to this file in `stores` package where it's also used.

Reply:
@maryia-deriv Ideally we shouldn't need the types anywhere as we just use `useStore` hook and types are inferred, But since we still might have some legacy codes which are not relying on the hook you might need the type, In that case you can just grab the type from `useStore` hook like below
```ts
type TActivePositions = ReturnType<typeof useStore>['portfolio']['active_positions']
```
Let me know If it doesn't work for your need 🙏🏻

Reply:
thank you, i got the idea, I've removed the export 👌🏻